browser/ipfs/ipfs_tab_helper.h

```diff
@@ -161,6 +168,8 @@ class IPFSTabHelper : public content::WebContentsObserver,
   void ShowBraveIPFSFallbackInfoBar(const GURL& initial_navigation_url);
   base::RepeatingCallback<void(const GURL&)>
       show_fallback_infobar_callback_for_testing_;
+  std::unique_ptr<BraveGlobalInfoBarManager> infobar_manager_;
```

Comment:
You have profile-scope object which is kept in web-contents-scope object. So it could be deleted when web contents is closed.